src/ioctls/system.rs

```diff
@@ -462,6 +504,33 @@ mod tests {
         assert_eq!(vm.run_size(), kvm.get_vcpu_mmap_size().unwrap());
     }
 
+    #[test]
+    #[cfg(any(target_arch = "aarch64"))]
+    fn test_create_vm_with_ipa_size() {
+        let kvm = Kvm::new().unwrap();
+        if kvm.check_extension(Cap::ArmVmIPASize) {
+            let vm = kvm.create_vm_with_ipa_size(40).unwrap();
+
+            // Test create_vmfd_from_rawfd()
+            let rawfd = unsafe { libc::dup(vm.as_raw_fd()) };
+            assert!(rawfd >= 0);
+            let vm = unsafe { kvm.create_vmfd_from_rawfd(rawfd).unwrap() };
+            assert_eq!(vm.run_size(), kvm.get_vcpu_mmap_size().unwrap());
```

Comment:
What are we trying to check here?

Reply:
Same thing in `create_vm` (see [here](https://github.com/rust-vmm/kvm-ioctls/blob/1072239e47c6b5dc6e143be8712ecde4ecc1347e/src/ioctls/system.rs#L457)). As this `create_vm_with_ipa_size` is an alternative function of `create_vm`, I personally think anything is covered in `create_vm` test should also be covered in `create_vm_with_ipa_size`.

Reply:
I don't think it is necessarily needed.

In that test we were checking that the `create_vmfd_from_rawfd` function works when the file descriptor passed is valid. Since this is already tested, we don't need to double check it now.

Reply:
Sure! I will remove this part after all your comments are addressed. Thanks.